src/main/scala/fable/Consumer.scala

```diff
@@ -175,6 +175,25 @@ class Consumer[F[_]: ContextShift: Monad: Sync, K, V] private[fable] (
       }.toMap
     }
 
+  /**
+    * Get metadata about partitions for all topics that the user is authorized to view. This method will issue a remote call to the server.
+Specified by:
+    *
+    * @see [https://kafka.apache.org/21/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#listTopics--]
+    */
+  def listTopics(): F[Map[Topic, List[Partition]]] =
+    for {
+      topics <- eval(_.listTopics())
+    } yield {
+      topics.asScala.map {
+        case ((topic, partitionInfos)) =>
+          val partitions: List[Partition] =
+            partitionInfos.asScala.toList.map(partitionInfo =>
+              Partition(Topic(topic), partitionInfo.partition))
+          (Topic(topic), partitions)
+      }.toMap
```

Comment:
Because `Partition` already has the `Topic`, does it make sense for this to be `Seq[Partition]` rather than `Map[Topic, List[Partition]`? What is the advantage of the `Map`?

Reply:
It might, we're not entirely clear on the use case for this(we picked it up because it looked relatively straight ahead). That being said returning the partitions makes it feel as though the method should be listPartitions. Perhaps that would be a more useful method?

Reply:
Given your above comment about matching the API and given how the existing KafkaClient works I would be tempted to keep the existing return type. 

https://kafka.apache.org/21/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#listTopics--

Reply:
Oh, I definitely misread the type at first. I think what you're doing makes total sense, because it maps to the `KafkaConsumer` method.